### PR TITLE
Add diet command to allow pet owners to put remarks for their pet's dietary concerns

### PR DESCRIPTION
**Pull request by soloplxya**
Refactored the Remark command to a Diet command.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DietCommand.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    //...
+    public static final String COMMAND_WORD = "diet";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Edits the diet of the pet identified "
+            + "by the index number used in the last pet listing. "
+            + "Existing diet will be overwritten by the input.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "d/ [diet]\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + "d/ Likes to swim.";
```
Comment (Tiffanylin21) [inline src/main/java/seedu/address/logic/commands/DietCommand.java:28]:
```suggestion
            + "d/ No seafood.";
```

File: src/test/java/seedu/address/logic/commands/DietCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+import seedu.address.model.UserPrefs;
+import seedu.address.model.pet.Diet;
+import seedu.address.model.pet.Pet;
+import seedu.address.testutil.PetBuilder;
+
+
+
+class DietCommandTest {
+
+    private static final String REMARK_STUB = "Some remark";
```
Comment (Tiffanylin21) [inline src/test/java/seedu/address/logic/commands/DietCommandTest.java:30]:
```suggestion
    private static final String DIET_STUB = "Some diet";
```

File: src/test/java/seedu/address/logic/commands/DietCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void execute_addDietUnfilteredList_success() {
+        Pet firstPerson = model.getFilteredPetList().get(INDEX_FIRST_PET.getZeroBased());
+        Pet editedPerson = new PetBuilder(firstPerson).withDiet(REMARK_STUB).build();
```
Comment (Tiffanylin21) [inline src/test/java/seedu/address/logic/commands/DietCommandTest.java:37]:
```suggestion
        Pet firstPet = model.getFilteredPetList().get(INDEX_FIRST_PET.getZeroBased());
        Pet editedPet = new PetBuilder(firstPet).withDiet(DIET_STUB).build();
```
just minor naming changes, can find and replace all 'person' with 'pet' and 'REMARK' with 'DIET' below too.

**Review by Tiffanylin21 — COMMENTED** [review]
Overall, just some minor naming issues 👍

**Comment by Dineshraj555** [comment]
LGTM. One point to think about. I realized you have a standalone command called diet. Since we are already adding diet as part of the add command and we already have an existing edit command to amend the diet accordingly. Do you think we should have diet as another standalone command?

**Pull request merged**